frontend/lib/pages/data-driven-onboarding.tsx

```diff
@@ -23,12 +23,14 @@ const PLACEHOLDER_IMG = 'frontend/img/96x96.png';
 
 const MAX_RECOMMENDED_ACTIONS = 3;
 
-const EFNYC_PRIORITY = 50;
-const VIOLATIONS_HIGH_PRIORITY = 45;
+const VIOLATIONS_PRIORITY = 50;
+const VIOLATIONS_HIGH_PRIORITY = 50;
 const COMPLAINTS_PRIORITY = 40;
-const VIOLATIONS_PRIORITY = 30;
-const RENT_HISTORY_PRIORITY = 20;
-const WOW_PRIORITY = 10;
+const EFNYC_PRIORITY = 30;
+const WOW_PRIORITY = 20;
+const RENT_HISTORY_PRIORITY = 10;
+
+const DISABLE_RECOMMENDATIONS = true;
```

Comment:
Maybe we should add a comment here just so future folks know why it's being put in there?  Eh, not a big deal either way.

Reply:
👍